Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/.template.config/template.json

```diff
@@ -55,7 +55,7 @@
         },
         "FrameworkShouldUseV1Dependencies": {
             "type": "computed",
-            "value": "(Framework == \"net48\" || Framework == \"net8.0\" || Framework == \"net7.0\" || Framework == \"net6.0\")"
+            "value": "(Framework == \"net48\" || Framework == \"net7.0\" || Framework == \"net6.0\")"
```

Comment:
@mattchenderson the PR description says that we are _"adding .NET 8 as a valid candidate"_ but we are removing .Net8 in this PR. What am I missing here?

Reply:
This is the list of TFMs that stay on V1. By removing it from the list, I have moved .NET 8 to use the V2 packages instead. .NET 9 has already been using V2 through its absence in this list.

Reply:
makes sense